src/message/message.go

```diff
@@ -67,7 +67,8 @@ func paragraph(format string, a ...any) string {
 }
 
 func debugPrinter(offset int, a ...any) {
-	printer := pterm.Debug.WithShowLineNumber(logLevel > 2).WithLineNumberOffset(offset)
+	showLines := logLevel == DebugLevel || logLevel == TraceLevel
```

Comment:
lines only showed on TraceLevel before so this should probably be <= Trace

Reply:
Done